.github/workflows/ci.yml

```
@@ -18,21 +17,19 @@ jobs:
     steps:
       - uses: actions/checkout@v6
 
+      - run: corepack enable
+
       - uses: actions/setup-node@v6
         with:
+          cache: yarn
           node-version: 22.x
           registry-url: https://registry.npmjs.org
 
-      - uses: actions/cache@v4
-        with:
-          path: ~/.npm
-          key: v1-${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: v1-${{ runner.os }}-node-
-
-      - run: npm ci
-      - run: npm run build
-      - run: npm run lint
-      - run: npm run test
+      - run: yarn install
```

```suggestion
      - run: yarn install --immutable
```

I think this is default in CI, but still brings me sanity to see it here explicitly